libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/RouteUtils.java

```diff
@@ -177,25 +179,33 @@ public boolean isValidRouteProfile(String routeProfile) {
    */
   @Nullable
   public BannerInstructions findCurrentBannerInstructions(LegStep currentStep, double stepDistanceRemaining) {
-    if (isValidStep(currentStep) && hasInstructions(currentStep.bannerInstructions())) {
-      int roundedDistanceRemaining = (int) stepDistanceRemaining;
+    if (isValidBannerInstructions(currentStep)) {
       List<BannerInstructions> instructions = new ArrayList<>(currentStep.bannerInstructions());
-      Iterator<BannerInstructions> instructionsIterator = instructions.iterator();
-      while (instructionsIterator.hasNext()) {
-        BannerInstructions instruction = instructionsIterator.next();
+      sortBannerInstructions(instructions);
```

Comment:
What about returning the sorted list instead of `void`? That way we could assign the sorted list in the previous line
```java
List<BannerInstructions> sortedInstructions = sortBannerInstructions(currentStep.bannerInstructions());
```
which IMO makes `findCurrentBannerInstructions` easier to read and understand. What do you think?